scripts/gh-pages.js

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable no-console,import/no-extraneous-dependencies */
+const ghpages = require('gh-pages')
+
+ghpages.publish('dist', err => {
+  console.warn(err)
```

Comment:
Should this not result in the script ending in failure? Otherwise we might not notice this during a build.